### DISCUSSION TITLE
Add OE, FNP, HUNO to TrackerHub

### DISCUSSION THREAD

**Discussion by nigiriemoji** [HDVinnie/TrackerHub#27049] [Ideas]
Hello

Could you please add the following 3 trackers to be monitored on TrackerHub? The first 2 run on UNIT3D I think

OnlyEncodes
https://onlyencodes.cc/

Fear No Peer
https://fearnopeer.com

Hawke Uno
https://hawke.uno/

Thank you

**Comment by HDVinnie** [comment]
done

**Comment by nigiriemoji** [comment]
Thank you much
